res/values/strings.xml

```diff
@@ -125,6 +125,9 @@
   <string name="ui_mark_as_read_label">Mark as read (emails)</string>
   <string name="ui_mark_as_read_desc">Whether to mark messages in Gmail as read or not.</string>
 
+  <string name="ui_sync_read_flag_label">Save read flag</string>
+  <string name="ui_sync_read_flag_desc">Whether to mark unread SMS and MMS in Gmail as unread or not.</string>
```

Comment:
Yeah, not sure about that as well. That's why I didn't added an option initially. Maybe we can change the description of "Mark as read" to "Whether to mark all messages (read and unread) in Gmail as read or not.". Or do you have a proposal?